tools/generator/AnDOGenerator.py

```diff
@@ -25,9 +25,13 @@
 
 METADATA_LEVEL_BY_NAME = {v: k for k, values in METADATA_LEVELS.items() for v in values}
 
+ESSENTIAL_CSV_COLUMNS = ['sub_id', 'ses_id']
+OPTIONAL_CSV_COLUMNS = ['tasks', 'runs']
+
+
 class AnDOData:
```

Comment:
should this be called AnDOSession instead of AnDOData (since, according to the docstring, it represents a session)?

Reply:
> @JuliaSprenger For info, I'm running into a problem at running time (i.e not with the tests), because "rules_set" has been renamed into "RULES_SET" (by me ;) ) recently... It's weird that the test don't fail because of this...
> 
> ```
> > python tools/generator/AnDOGenerator.py 
> Traceback (most recent call last):
>   File "tools/generator/AnDOGenerator.py", line 15, in <module>
>     from ando.rulesStructured import rules_set
> ImportError: cannot import name 'rules_set' from 'ando.rulesStructured' (/home/takerkart/work/python/PR_reviews/AnDO/ando/rulesStructured.py)
> ```

Yes, I think I have the same fix in one of the other branches probably. But it's good to have it fxed here, too.

Reply:
I was not sure about the best term to use here, since it's not only representing what AnDO specifies at the Session level, but also all  infos on other levels. How about keeping the name `AnDOData` but extending the docstring?

Reply:
ok for me! it's not a big deal anyway...